### DISCUSSION TITLE
Best practice?

### DISCUSSION THREAD

**Discussion by hurovroman** [hurovroman/eCommerce-NextJS-Sanity#4] [General]
Another one question;
What is the best practice to Storing JWT in Local Storage?

**Comment by mythofmidas** [comment]
Yeah, Here are answers;
1. Use HTTPS: Always use HTTPS to ensure the data transmitted between the client and server is encrypted.
2. Token Expiry: Ensure your JWT tokens have an expiry time and refresh them as necessary.
3. Secure Storage: Prefer HTTP-only cookies for storing sensitive tokens to mitigate XSS risks.
